testing/sdk_install.py

```diff
@@ -32,6 +32,26 @@
 _installed_service_names = set([])
 
 
+def get_package_json(package: str,
+                     version: str=None,
+                     options: dict=None) -> dict:
+    """Get the Marathon app JSON for a given package with the desired
+    options.
+
+    Args:
+        package: Package name
+        version: Package version
+        options: Package installation options
+
+    Returns: Marathon app def
+
+    """
+    pkg_version = dcos.packagemanager.CosmosPackageVersion(
```

Comment:
As Evan said, if at all possible we should hit the CLI rather than the underlying library as the CLI team plans to drop the library.